packages/playground/cli/src/run-cli.ts

```diff
@@ -142,10 +142,11 @@ export async function parseOptionsAndRunCLI() {
 				type: 'boolean',
 				default: false,
 			})
-			.option('quiet', {
+			.option('verbosity', {
```

Comment:
Let's still accept the `--quiet` option and translate it into verbosity to avoid breaking BC. We can just hide it from the help message (`hidden: true` I think?)